src/skottie-player.ts

```diff
@@ -17,16 +64,111 @@ export class SkottiePlayer extends LitElement {
     }
   `;
   }
+  
+  /**
+   * The title of the json file. This is displayed above the animation.
+   */
+  @property({type: String})
+  title = '';
+
+  /**
+   * The content of the json file.
+   */
+  @property({type: Object})
+  content:lottieFile = {};
+
+  /**
+   * The URL/path to the json file.
+   */
+  @property({type: String})
+  fileURL = '../samples/animations/gear.json';
```

Comment:
Will this file path be changed once we add the feature to load different json files?

Reply:
This was a placeholder for when I was testing on local files -- it gets changed to the file's url when the user uploads a json.